### PR TITLE
Document a few function calls and the boost_effect module

### PR DESCRIPTION
**Pull request by JaceCear**

### CODE REVIEW

File: src/game/stage/results.c [hunk 1]
```diff
@@ -297,8 +298,13 @@ u16 CreateStageResults(u32 courseTime, u16 ringCount, u8 spRingCount)
 
     return (u16)outro->unk16C;
 }
+#endif
 
-void Task_UpdateStageResults(void)
+// (98.99%) https://decomp.me/scratch/19nrW
+// NOTE: Only non-matching thing is TasksDestroyAll() not
+//       getting initialized like in the original.
+NONMATCH("asm/non_matching/game/stage/outro/Task_UpdateStageResults.inc",
```
Comment (freshollie) [inline src/game/stage/results.c:306]: Umm?

File: sym_iwram.txt [hunk 2]
```diff
@@ -73,10 +73,10 @@ build/sa2/src/core.o(.data);
 . = 0x000055E0; gNewInputCounters = .;
 . = 0x00005660; gWater = .;
 . = 0x000056A4; gUnknown_030056A4 = .;
-. = 0x000056B0; gUnknown_030056B0 = .;
-. = 0x000057B0; gUnknown_030057B0 = .;
-. = 0x00005830; gUnknown_03005830 = .;
-. = 0x00005834; gUnknown_03005834 = .;
+. = 0x000056B0; sPlayerStateBuffer = .;
```
Comment (freshollie) [inline sym_iwram.txt:76]: Consider using the `boost_mode_particles` file to generate these

File: asm/non_matching/game/stage/outro/Task_UpdateStageResults.inc
[the remark this answers is not in the document]
Comment (JaceCear) [reply asm/non_matching/game/stage/outro/Task_UpdateStageResults.inc:1]: *SIGH*
Don't you love it when Visual Studio screwes you over? D=

File: sym_iwram.txt (hunk 2 above)
Comment (JaceCear) [reply sym_iwram.txt]: Done!

**Review by freshollie — APPROVED** [review]

**Pull request merged**